content/docs/guide-development.md

```diff
@@ -117,7 +117,51 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+To support the development of workflows in Renku, a set of dedicated funcitonailities, provided as Renku plugins, are made available. In particular those aim to:
 
+* Offer an interactive (and non) visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
+* Create dedicated annotations for the execution of `astroquery` functions and add those to the project Knowledge Graph (`renku-aqs-annotation` plugin)
+
+Those two set of features are provided within different plugins.
+
+### Visualizing project Knowledge Graph with `renku-graph-vis` plugin
+
+Starting from the knowledge graph, this is used to generate an interactive graphical representation of the graph itself.
+
+The plugin provides two CLI commands:
+  * `display` to generate a representation of the graph over an output image
+  * `show-graph` to start an interactive visualization of the graph over the browser
+
+In addition, the plugin offers a dynamic visualization of the graph during an interactive renku session.
+
+The primary benefit introduced is the ability to have a live overview of the ongoing development, during an interactive Renku session. This can be seen within the animation below, where the graph is automatically updated with information about the execution of a notebook upon its completion.
```

Comment:
```suggestion
The primary benefit introduced is the ability to have a live overview of the ongoing development within an interactive Renku session. This can be seen within the animation below, where the graph is automatically updated with information about the execution of a notebook upon its completion.
```